cmd/kubeadm/app/cmd/config.go

```diff
@@ -93,6 +93,7 @@ func NewCmdConfig(out io.Writer) *cobra.Command {
 // TODO: Make it possible to print the defaults for the componentconfig API objects, and default to printing them out as well
 func NewCmdConfigPrintDefault(out io.Writer) *cobra.Command {
 	apiObjects := []string{}
+	componentConfig := false
```

Comment:
the above TODO comment suggested to default this to `true`, was there a change in plan?

Reply:
Got that one reversed. Will fix it.